components/Header.js

```diff
@@ -11,4 +11,37 @@
 // Use your function to create a header
 // and append it to the DOM inside the div.header-container
 
-function Header() {}
+
+function Header() {
+
+   const div =document.createElement('div');
+   const span=document.createElement('span');
+   const spantemp=document.createElement('span');
+   const h1=document.createElement('h1');
+   
+  
+
+
+div.classList.add('header');
```

Comment:
Please indent lines 25 to 41 to be aligned with line 20 since this is still inside the `Header()` function. That way it is easy to see what is the body of the function.